README.md

```diff
@@ -26,8 +26,9 @@ What is Bitcoin Unlimited?
 
 Bitcoin Unlimited is an implementation of the Bitcoin client software that is based on Bitcoin Core.
 However, Bitcoin Unlimited has a very different philosophy than Core.
-It follows a philosophy and is administered by a formal process described in the [Articles of Federation](https://www.bitcoinunlimited.info/resources/BUarticles.pdf).
-In short, we believe in market driven decision making, emergent consensus, and in giving our users choices.
+
+It follows a philosophy and is administered by a formal process described in the [Articles of Federation](http://www.bitcoinunlimited.info/resources/BUarticles.pdf).
```

Comment:
this commit switch back the url to using http scheme rather than https. It does no harm cause the site automatically redirect plain traffic to ssl one, still worth pointing out though.